describe_log_dirs_request.go

```diff
@@ -82,6 +82,10 @@ func (r *DescribeLogDirsRequest) headerVersion() int16 {
 	return 1
 }
 
+func (r *DescribeLogDirsRequest) isValidVersion() bool {
+	return r.Version == 0
```

Comment:
Version 1 is just response throttling behaviour change so I think we support that too.